tools/genevergreen/generate/generate.go

```diff
@@ -70,11 +70,19 @@ var (
 	}
 )
 
-func newDependency(toolName, os, serverVersion, repo string) shrub.TaskDependency {
-	return shrub.TaskDependency{
-		Name:    fmt.Sprintf("push_%s_%s_%s_stable", toolName, newOs[os], repo),
-		Variant: fmt.Sprintf("release_%s_publish_%s", toolName, strings.ReplaceAll(serverVersion, ".", "")),
+func newDependency(toolName, os, serverVersion, repo string) []shrub.TaskDependency {
+	var deps []shrub.TaskDependency
+	architectures := Distros[newOs[os]].architectures
+
+	for _, arch := range architectures {
```

Comment:
this test is currently written to test amd64 version of the bins, depending on the ARM publishing won't do much for these test

Reply:
updated